blocks-helpers/focusTrap.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Traps focus in a menu.
+ * @function
+ * @param {Element} focusTrapContainer - the wrapper we wish to trap a user within
+ * @return a cleanup function
+ */
+
+export const createFocusTrap = (focusTrapContainer) => {
```

Comment:
We can still tab into the skip link while focus is trapped. Activating the skip link doesn't seemingly do anything.

What do you think about either not rendering the skip link when the menu panel is open, or upon activating the skip link the menu panel is closed and the user is taken to the main content?

I'm leaning towards the second option, unless there is another option you've got in mind?

Reply:
Oh excellent catch!

Reply:
I've also fixed it so the skip link is the top layer. 
<img width="411" alt="Screenshot 2025-05-20 at 1 59 01 PM" src="https://github.com/user-attachments/assets/ec92ed05-ca9b-421b-8790-7f5b809fbb16" />